pages/api/trees/gdk.ts

```diff
@@ -0,0 +1,77 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+const SUPABASE_ANON_KEY = process.env.SUPABASE_ANON_KEY
+const SUPABASE_API_URL = process.env.SUPABASE_API_URL
+/**
+ * Small wrapper to fetch the primary key id
+ * from the giessdenkiez.de database API by looking it up
+ * based on the gmlid used in qtree.
+ * Used to create a link in the UI to direct users to the right tree
+ * in the giessdenkiez.de.
+ *
+ * FInal link in giessdenkiez.de looks like this:
+ * `https://www.giessdenkiez.de/tree/_1234567`
+ * @example
+ * fetch('/api/trees/gdk?gmlid=00008100:000be766')
+ * // returns { data: { id: "_1234567", gmlid: '00008100:000be766' } }
+ * @example
+ * $ curl "http://localhost:3000/api/trees/gdk?gmlid=00008100:000be766"
+ * > { "data": { "id": "_1234567", "gmlid": "00008100:000be766" } }
```

Comment:
I've tried this request and I believe the examples are missing brackets. Isn't it that `data` always returns an _array_ of results?

```
{"data":[{"id":"_1234567","gmlid":"00008100:000be766"}]}
```

Reply:
Yes. You are right. Fixed